tree/ntupleutil/v7/test/ntuple_inspector.cxx

```diff
@@ -472,26 +494,74 @@ TEST(RNTupleInspector, PageSizeDistribution)
       }
    }
 
+   auto ntuple = ROOT::Experimental::RNTupleReader::Open("ntuple", fileGuard.GetPath());
+   ntuple->PrintInfo(ROOT::Experimental::ENTupleInfo::kStorageDetails);
```

Comment:
This will print to stdout, won't it? It would be nice to keep stdout clean so that the GTest output doesn't get cluttered. It looks like the statement can be removed or, if needed, it can be redirected to a custom stream.

Reply:
Thanks for pointing this out! It was leftover from some debugging and I forgot to remove it. It's gone now (along with the `inspector->PrintColumnTypeInfo()`)